in_toto/verifylib.py

```diff
@@ -240,6 +242,84 @@ def verify_layout_expiration(layout):
     raise LayoutExpiredError("Layout expired")
 
 
+def substitute_parameters(layout, parameter_dictionary):
+  """
+  <Purpose>
+    This function is a transitionary measure for parameter substitution (or
+    any other solution defined by the in-toto team). As of now, it acts as 
+    a very simple replacement layer for python-like parameters
+
+  <Arguments>
+    layout:
+            The Layout object to process.
+
+      parameter_dictionary:
+            A dictionary containing key-value pairs for substitution.
+
+  <Exceptions>
+    securesystemslib.exceptions.FormatError:
+      if the parameter dictionary is malformed.
+
+    KeyError:
+      if one of the keys in the parameter dictionary are not present for
+      substitution
+
+  <Side Effects>
+    The layout object will have any tags replaced with the corresponding
+    values defined in the parameter dictionary.
+  """
+  in_toto.formats.PARAMETER_DICTIONARY_SCHEMA.check_match(parameter_dictionary)
+
+  for step in layout.steps:
+
+    new_material_rules = []
+    for rule in step.expected_materials:
+      new_rule = []
+      for stanza in rule:
```

Comment:
Why did you choose the name `stanza`?  Is this terminology used when dealing with variable substitutions?

Reply:
No, but I couldn't think of anything better :confounded: